cli/src/lib/npm/npmProjectUtils.js

```diff
@@ -74,6 +74,22 @@ export async function findPackageJsonPath(pathStr: string): Promise<string> {
   return path.join(pkgJsonPathStr, 'package.json');
 }
 
+async function getInstalledFlowPackageJsonData(pathStr: string): Promise<PkgJson | null> {
+  const flowBinPackagePath = 'node_modules/flow-bin/package.json';
```

Comment:
I see there was a good suggestion in #1282 to find the package.json.

Reply:
Actually I believe that if it's a monorepo using `require.resolve` might be the only solution?

Reply:
Let me try that, my only experience with mono-repos is yarn workspace which pulls node_modules to the top. Is this a case when node_modules would be embedded inside a nested dir?

Reply:
I remember awhile back I had some issues with `require.resolve` and started using the `resolve` package instead in a lot of my codemod tools.  I think the issue may have been that `require.resolve` didn't work in code running inside of VSCode extensions, but `resolve` did?

Reply:
I don't think we can use `require()` or `require.resolve()` on a `flow-bin`. The way the code currently looks up package.json uses an absolute path with `cwd`. There is one area that does a require with a relative [path](https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/npm/npmProjectUtils.js#L125) and this is relative to where the script is run, I don't think it has the ability to import/require dependencies from another project, especially since it's generally installed globally. That's probably why this script generally leverages `fs.read`

If I changed this to require or require.resolve it would run inside the `flow-typed` project not your project right?

Reply:
It seems to work:

Reply:
@Brianzchen I don't have much experience with mono-repos really, so please do what you think is best.